components/homme/src/theta-l/config.h.cmake.in

```diff
@@ -63,8 +63,8 @@
 /* Whether to use OpenMP4 */
 #cmakedefine OMP4
 
-/* Whether this is a build for CUDA (used mostly for f90-vs-cxx bfb math functions impl) */
-#cmakedefine CUDA_BUILD
+/* Whether this is a build for CUDA/HIP (used mostly for f90-vs-cxx bfb math functions impl) */
```

Comment:
The comment about BFB should be removed since this symbol is used for tons of stuff other than that.